mne/io/edf/edf.py

```diff
@@ -511,16 +481,529 @@ def _get_edf_info(fname, stim_channel, annot, annotmap, eog, misc, exclude,
+                            datetime.timedelta(birthday * pow(2, -32) - 367))
+            patient['birthday'] = birthday
+            if patient['birthday'] != datetime.datetime(1, 1, 1, 0, 0):
+                today = datetime.datetime.today()
+                patient['age'] = today.year - patient['birthday'].year
+                today = today.replace(year=patient['birthday'].year)
+                if today < patient['birthday']:
+                    patient['age'] -= 1
+            else:
+                patient['age'] = None
+
+            header_nbytes = np.fromfile(fid, np.uint16, 1)[0] * 256
+
+            fid.seek(6, 1)  # 6 bytes reserved
+            equipment = np.fromfile(fid, np.uint8, 8)
+            ip = np.fromfile(fid, np.uint8, 6)
+            patient['headsize'] = np.fromfile(fid, np.uint16, 3)
+            patient['headsize'] = np.asarray(patient['headsize'], np.float32)
+            patient['headsize'] = np.ma.masked_array(
+                patient['headsize'],
+                np.equal(patient['headsize'], 0), None).filled()
+            ref = np.fromfile(fid, np.float32, 3)
+            gnd = np.fromfile(fid, np.float32, 3)
+            n_records = np.fromfile(fid, np.int64, 1)[0]
+
+            # record length in seconds
+            record_length = np.fromfile(fid, np.uint32, 2)
+            if record_length[0] == 0:
+                record_length[0] = 1.
+                warn('Header information is incorrect for record length. '
+                     'Default record length set to 1.')
+
+            nchan = np.fromfile(fid, np.uint16, 1)[0]
+            fid.seek(2, 1)  # 2bytes reserved
+
+            # Channels (variable header)
+            channels = list(range(nchan))
+            ch_names = [fid.read(16).decode().strip(' \x00')
+                        for ch in channels]
+            exclude = [ch_names.index(idx) for idx in exclude]
+
+            fid.seek(80 * len(channels), 1)  # reserved space
+
+            """The Physical Dimensions are encoded with a 16 bit integer
+            number, as a combination of 'unit code' + order of magnitude.
+            The list of possible codes can be found in the following
+            documents:
+            - Units codes :
+            https://sourceforge.net/p/biosig/svn/HEAD/tree/trunk/biosig/doc/units.csv
+            - Decimal factors codes:
+            https://sourceforge.net/p/biosig/svn/HEAD/tree/trunk/biosig/doc/DecimalFactors.txt
+
+            For example, the physical dimension 'uV' (microvolt) is
+            encoded as 4256 (Volt) + 19 (micro) = 4275. The code 512
+            stands for dimensionless.
+            """  # noqa
+            fid.seek(6 * len(channels), 1)  # phys_dim, obsolete
+            units = np.fromfile(fid, np.uint16, len(channels)).tolist()
+            unitcodes = np.array(units[:])
+            include = list()
+            for i, unit in enumerate(units):
+                if unit == 4275:  # microvolts
```

Comment:
there are starting to be a lot of constants. Can you add a `constants.py` and add these in there? See how we do it for other readers like FIF, CTF, KIT, etc. It makes things more readable and DRY without needing these comments.